### DISCUSSION TITLE
seems like all the stackblitz examples on the Demo page are not working

### DISCUSSION THREAD

**Discussion by glickid** [ng-select/ng-select#2488] [Q&A]
seems like all the stackblitz examples on the[ Demo page](https://ng-select.github.io/ng-select/#/data-sources) are not working
e.g.[ this stackblitz](https://stackblitz.com/run?file=src%2Fdata-source-backend-example.component.html)

![image](https://github.com/user-attachments/assets/dafd29bb-3728-4ce0-a2a5-f17275f3fb5c)

please advise

**Comment by amcardwell** [comment]
Bump. I would appreciate it if this was fixed :)

**Comment by pavankjadda** [comment]
The issue has been fixed
